docs/guides/private-python-packages.rst

```diff
@@ -0,0 +1,108 @@
+Using Private Python Packages with Read the Docs for Business
+=============================================================
+
+Read the Docs uses :doc:`PIP <pip:index>` to install your Python packages,
+see :doc:`/guides/specifying-dependencies`.
+
+Installing packages from a Git repository
+-----------------------------------------
+
+PIP supports installing packages from a :ref:`Git repository <pip:vcs support>` using the form:
```

Comment:
Despite my comment above, I'm ok with an initial capital "Pip" here as the start of a sentence.